backend/src/threads/services/review_service.py

```diff
@@ -40,11 +45,20 @@ def create(self, request: Request, thread: ReviewCreate, db: Session) -> Respons
         return Response(status_code=201, content="Review successfully created.")
 
     def get_thread_by_id(self, thread_id: uuid.UUID, db: Session) -> ReviewSchema:
+        cache_review = cache.get_value(f"thread:{thread_id}")
```

Comment:
Im not sure of this convention, we can consider keeping keys like consts and import itd, for example as enums.
For example we can keep it as enum liken 
```
class CacheKeys(Enum):
     THREAD = 'thread:'

```

and then we can import it and write like 
`CacheKeys.THREAD.value+ thread_id`